packages/zosfiles/src/methods/copy/Copy.ts

```diff
@@ -84,4 +92,239 @@ export class Copy {
             throw error;
         }
     }
+
+
+    /**
+     * Copy the contents of a dataset from one LPAR to another LPAR
+     *
+     * @param {AbstractSession}                 sourceSession  - Source z/OSMF connection info
+     * @param {IDataSet}                        toDataSet      - The data set to copy to
+     * @param {ICrossLparCopyDatasetOptions}    targetOptions  - Options for target file and connection
+     * @param {IGetOptions}                     sourceOptions  - Options for source file
+     * @param {IDataSetOptions}                 options        - Common options
+     * @param {IHandlerResponseConsoleApi}      console        - Command console object
+     *
+     * @returns {Promise<IZosFilesResponse>} A response indicating the status of the copying
+     *o
+     * @throws {ImperativeError} Data set name must be specified as a non-empty string
+     * @throws {Error} When the {@link ZosmfRestClient} throws an error
+     *
+     */
+    public static async dataSetCrossLPAR(
+        sourceSession: AbstractSession,
+        { dsn: toDataSetName, member: toMemberName }: IDataSet,
+        options: ICrossLparCopyDatasetOptions,
+        sourceOptions: IGetOptions,
+        targetSession: AbstractSession
+    ): Promise<IZosFilesResponse> {
+        ImperativeExpect.toBeDefinedAndNonBlank(options["from-dataset"].dsn, "fromDataSetName");
+        ImperativeExpect.toBeDefinedAndNonBlank(toDataSetName, "toDataSetName");
+
+
+        try {
+            let sourceDataset = options["from-dataset"].dsn;
+            const sourceMember  = options["from-dataset"].member;
+            let   sourceDataSetObj: IZosmfListResponse;
+            let   targetDataset = toDataSetName;
+            const targetMember  = toMemberName;
+            let targetDataSetObj: IZosmfListResponse;
+            let targetFound: boolean = false;
+            let targetMemberFound: boolean = false;
+            let overwriteTarget: boolean = options.replace;
+
+            /**
+             * If no console is passed and no overwrite is passed then set to owerwrite to false
+             */
+            if(console == undefined && overwriteTarget == undefined){
```

Comment:
I believe the `console == undefined` check is left behind from when this method accepted a `console` parameter.

We should remove it now - it will always evaluate to `false` because the JS console API is always defined.

Reply:
This check has been removed.